lib/src/style/mapbox_styles.dart

```diff
@@ -36,6 +36,12 @@ class MapboxStyles {
   /// improve the style.
   static const String SATELLITE_STREETS =
       "mapbox://styles/mapbox/satellite-streets-v12";
+
+  /// NOT FOR PRODUCTION USE. An experimental version of the Mapbox Standard style.
+  /// This style is used for testing new features and changes to the Mapbox Standard style. The style may change or be removed at any time.
+  @experimental
```

Comment:
This can be removed, I've just left it in for now for convenience when reviewing.